qemu/tests/cfg/qmp_event_notification.cfg

```diff
@@ -91,3 +91,10 @@
                 - qmp_none:
                     extra_params += " -watchdog i6300esb -watchdog-action none "
                     action_check = "none"
+        - qmp_guest_panicked:
+            no Windows
+            enable_pvpanic = yes
+            ioport_pvpanic = 0x505
```

Comment:
only for q35 and pc,  no need for ppc,

Reply:
@yanan-fu update again and pls review it

(17/18) smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_guest_panicked.from_guest.q35: PASS (107.44 s)

(17/18) smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_guest_panicked.from_guest: PASS (107.23 s)

Thanks
Jing